docs/api/users-change-user-email.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+---
+
+# Update user email
+
+The Patch request updates the user's email address but does not change the other parameters in the [`user`](user) object. 
+
+**Note** : *After creating a user, a unique identifier (ID) is assigned. Provide the ID for the user in the REST URL.* 
+
+## URL
+
+```shell
+{server_url}/users/{id}
+```
+## Method
+PATCH
+
+
+## Params
+
+| Parameter name | Type | Description |
+| -------------- | ------ | ------------ |
+| `email` | string | The user's new email address that will replace the address in the current record. |
+| `id` | number | The user's unique record ID. |
+
+## Request headers
+
+None
+
+## Request body
+
+{
+	"email":"marty2@example.com"
+} 
```

Comment:
This example is different from what is listed in **Params** above. Is that intentional?
If `id` is is optional, then that should be mentioned in the **Params** section.